### DISCUSSION TITLE
Ability to sync camera positions across non-legacy 3D panels

### DISCUSSION THREAD

**Discussion by bdemiroz-nvidia** [foxglove/community#591] [Q&A] [answered]
With the legacy 3D panels we had the ability to sync cameras across panels. With the new 3D panels I couldn't find that feature. Am I missing something? If not, what do we need to do to request that feature back?

Also see:
- [Legacy 3D panel doc](https://foxglove.dev/docs/studio/panels/3d-legacy) mentions syncing
- [New 3D panel doc](https://foxglove.dev/docs/studio/panels/3d)

**Comment by foxymiles** [comment, marked as answer]
This is now available in settings, under scene > sync camera:

<img width="409" alt="Screenshot 2022-12-29 at 10 58 09 AM" src="https://user-images.githubusercontent.com/93935560/209893656-60b0cc4a-c5ed-4e66-89f7-ef252dc255ee.png">

**Reply by bdemiroz-nvidia** [reply]
This worked, thanks a lot!
